src/world.rs

```diff
@@ -646,3 +660,67 @@ impl Default for World {
         }
     }
 }
+
+#[cfg(feature="serialize")]
+/// Structure used to serialize a world using a component group.
+pub struct WorldSerializer<'a, G> {
```

Comment:
I don't really mind either way since it would still be possible if it was a function instead of a struct.